src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/InstallAppsTask.groovy

```diff
@@ -141,7 +215,7 @@ class InstallAppsTask extends AbstractServerTask {
             break
         default:
             logger.info(MessageFormat.format(("Loose application configuration is not supported for packaging type {0}. The project artifact will be installed as an archive file."),
-                    project.getPackaging()))
+                    project.getPackagingType()))
```

Comment:
Change this to `getPackagingType()` instead of project `project.getPackagingType()`